src/DotNetty.Transport/Channels/Sockets/SocketDatagramChannel.cs

```diff
@@ -97,7 +97,9 @@ protected override void DoClose()
         {
             if (this.TryResetState(StateFlags.Open | StateFlags.Active))
             {
-                this.Socket.Shutdown(SocketShutdown.Both);
+#if !NETSTANDARD1_3                
```

Comment:
Dispose calls Close internally (if it wasn't called before) so no need to add explicit Close here.

Reply:
Agreed.